components/item/AwesomeSwiper.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <swiper
+    ref="awesome"
+    class="swiper"
```

Comment:
we're not using this class, but might be useful to keep around as we start applying more custom styling ourselves.